test/new-e2e/language-detection/language_detection_test.go

```diff
@@ -0,0 +1,111 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package languagedetection
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+)
+
+// testLocally should be set to true if running the test from a developer machine (i.e. not CI)
+// This sets the test suite to use the hardcoded agent version `versionStr`
+const testLocally = false
+
+const versionStr = "7.48.0~rc.1-1"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Would it make sense to have this defined on your local profile ? Maybe by default target the latest RC when running locally ?

Reply:
What do you mean by local profile? Also targeting the latest RC by default would be nice, but do you have any ideas how to do that?

Reply:
Local profile is the `~/.test_infra_config.yaml` that we use to store and parse local configuration variables. On the CI we use a mix of environment variables and config parameters passed through the invoke task that runs tests. 

I'll check for the best way to have local test runs using latest beta, you can merge this as is for the moment, we'll take care of running by default on latest master or latest beta, and possibly adding a configuration entry in the local configuration file for it !